tests/mapped_struct.py

```diff
@@ -974,6 +974,8 @@ def gen_values(self, *p, **kw):
         str_ = str
         for k, v in super(ApproxStringIdMultiMapperTest, self).gen_values(*p, **kw):
             yield str_(k), v
+        for k, v in super(ApproxStringIdMultiMapperTest, self).gen_values(*p, **kw):
+            yield k, v
```

Comment:
Maybe use a separate test for this. I'm not fond of mixing the two key sets in the same test, it may muddy the test's results.